gallery/src/components/NxTabs/NxTabsModalExample.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, {useState} from 'react';
+
+import {
+  NxTabs,
+  NxTabList,
+  NxTab,
+  NxTabPanel,
+  NxModal,
+  NxFontAwesomeIcon
+} from '@sonatype/react-shared-components';
+
+import {faAngry} from '@fortawesome/free-solid-svg-icons';
+
+const NxTabsModalExample = () => {
+  const [showModal, setShowModal] = useState(false);
+  const modalCloseHandler = () => setShowModal(false);
+
+  const [activeTabId, setActiveTabId] = useState<number>(0);
+
+  function handleTabSelect(index: number) {
+    setActiveTabId(index);
+  }
+
+  return (
+    <>
+      <button onClick={() => setShowModal(true)} className="nx-btn">Open Modal</button>
+      { showModal &&
+        <NxModal id="nx-modal-simple-example" onClose={modalCloseHandler} className="nx-modal--wide">
+          <header className="nx-modal-header">
+            <h2 className="nx-h2">
+              <NxFontAwesomeIcon icon={faAngry} />
+              <span>Modal header</span>
+            </h2>
+          </header>
+          <div className="nx-modal-content">
+            <NxTabs activeTab={activeTabId} onTabSelect={handleTabSelect}>
```

Comment:
I'd check with @dturner32 about the way this example ended up looking.  Feels odd to me

Reply:
> I'd check with @dturner32 about the way this example ended up looking. Feels odd to me

It is respecting the correct `24px` internal padding of `nx-modal-content`, I think it just looks a little off because of the space above the tabs that appears empty, but isn't because that's where the `:focus` rectangle goes.
<img width="1022" alt="Screen Shot 2020-09-29 at 2 42 08 PM" src="https://user-images.githubusercontent.com/1991443/94601775-f88cea80-0261-11eb-8d7d-a7e3946c5b04.png">
<img width="1022" alt="Screen Shot 2020-09-29 at 2 40 22 PM" src="https://user-images.githubusercontent.com/1991443/94601779-f9258100-0261-11eb-91ac-8b00f7e36d2e.png">

Reply:
> It is respecting the correct 24px internal padding of nx-modal-content, I think it just looks a little off because of the space above the tabs that appears empty, but isn't because that's where the :focus rectangle goes.

Right, it's certainly respecting the rules we have so far, it's just a question of whether the result looks "good", especially when none of the tabs are focused.
![image](https://user-images.githubusercontent.com/3630091/94602590-11e26680-0263-11eb-91b9-9f188911353b.png)